src/utils/fetchAPI.ts

```diff
@@ -0,0 +1,6 @@
+export default async function games() {
```

Comment:
```suggestion
export default async function getGames() {
```
Try to describe what the function actually does

Reply:
Damn it, I forgot to push my last commit, I already did that and other suggested stuff, thanks though :(